packages/shared/src/utils/date/date-time.ts

```diff
@@ -274,3 +275,24 @@ export const convertTimeFormat = (time: string) => {
     const time_suffix = `${Number(time_hour) >= 12 ? 'pm' : 'am'}`;
     return `${formatted_time} ${time_suffix}`;
 };
+
+/**
+ *  Get a formatted time since a timestamp.
+ * @param  {Number} timestamp in ms
+ * @return {String} '10s ago' or '1m ago' or '1h ago' or '1d ago' or '1 Jan 2019'
```

Comment:
we don't have '1d ago' logic in the code 😅

Reply:
@balakrishna-deriv yeah, i took the logic from bot, and'1d ago' was extra in the comment. But I just checked with PO, and it turns out we won't need the full date here, only '1d ago' :) i'll fix it